doc/source/whatsnew/v0.18.2.txt

```diff
@@ -132,3 +132,4 @@ Bug Fixes
 
 - Bug in ``NaT`` - ``Period`` raises ``AttributeError`` (:issue:`13071`)
 - Bug in ``Period`` addition raises ``TypeError`` if ``Period`` is on right hand side (:issue:`13069`)
+- Bug in ``__call__`` of ``EngFormatter`` raises ``InvalidOperation`` if NaN is passed (:issue:`11981`)
```

Comment:
make this more user friendly. IOW a bug that would prevent NaN's from formatting.